cdap-ui/app/cdap/components/Footer/index.js

```diff
@@ -17,16 +17,45 @@
 import React from 'react';
 import { Theme } from 'services/ThemeHelper';
 import If from '../If';
-import { objectQuery } from 'services/helpers';
+import { isNilOrEmptyString, objectQuery } from 'services/helpers';
+import NamespaceStore from 'services/NamespaceStore';
 require('./Footer.scss');
 
 export default function Footer() {
   const footerText = Theme.footerText;
   const footerUrl = Theme.footerLink;
   // 'project-id-30-characters-name1/instance-id-30-characters-name';
   const instanceMetadataId = objectQuery(window, 'CDAP_CONFIG', 'instanceMetadataId');
+  const [selectedNamespace, setSelectedNamespace] = React.useState(
+    NamespaceStore.getState().selectedNamespace
+  );
+  React.useEffect(() => {
+    const sub = NamespaceStore.subscribe(() =>
+      setSelectedNamespace(NamespaceStore.getState().selectedNamespace)
```

Comment:
nit: `getCurrentNamespace` is a function that we use across UI. Can you replace this with the function. It is essentially the same.